images/daos-client-image.json

```diff
@@ -1,7 +1,7 @@
 {
-    "variables": {
-        "DAOS_VERSION": "2.0.0"
-    },
+  "variables": {
+    "daos_version": "{{env `DAOS_VERSION`}}"
+  },
```

Comment:
@markaolson    How is the value for DAOS_VERSION set?  We don't wan Users messing around with this dev feature.   We want our users on Intel/GCP Validated machines images not a choose your own adventure.

Reply:
The version is no longer hard coded in each of the packer templates. 
The default version is now in a single file images/daos_version.sh.  
That file contains the default version used when no version is specified in the -v --version option of images/build_images.sh